script.js

```diff
@@ -1,8 +1,8 @@
 let projectData = [
```

Comment:
Please don't change the existing styling.
Are you using any kind of formatter, if using disable it while saving this file.
It is changing in every line of the project.
Only required changes need to be reflected and yours is changing everything.
fix it.
Also squash the commits, if possible if you don't know
comment here.

Reply:
Yes, earlier i was using formatter i.e why i made this much commits 
Now I have corrected it.